app/assets/stylesheets/config/utilities.css.scss

```diff
@@ -0,0 +1,11 @@
+.u--hide-on-mobile {
```

Comment:
I don't think this is used anywhere? But if you just added it here to have the opposite of show-on-mobile then ignore me :)

Reply:
yep, I used it before, stopped but left so it's there to use. but I don't usually do that. actually, I'll remove :P